Osiris/Hacks/Backtrack.cpp

```diff
@@ -31,13 +32,14 @@ void Backtrack::update(FrameStage stage) noexcept
 
             Record record{ };
             record.origin = entity->getAbsOrigin();
+            record.head = entity->getBonePosition(8);
             record.simulationTime = entity->simulationTime();
 
             entity->setupBones(record.matrix, 256, 0x7FF00, memory->globalVars->currenttime);
 
             records[i].push_front(record);
-
-            while (records[i].size() > 3 && records[i].size() > static_cast<size_t>(timeToTicks(static_cast<float>(config->backtrack.timeLimit) / 1000.f)))
+            int timeLimit = config->backtrack.timeLimit; if (timeLimit > 200) timeLimit = 200;
```

Comment:
Probably not needed if you use `config->backtrack.fakeLatency ? 400 : 200` in GUI.cpp

Reply:
If ImGui can't change number lower than 200 after disabling fakeLatency, need to use
```cpp
config->backtrack.timeLimit = config->backtrack.fakeLatency ? config->backtrack.timeLimit : 200
```

Reply:
`config->backtrack.timeLimit = config->backtrack.fakeLatency ? config->backtrack.timeLimit : 200`
just locks the timelimit to 200 when fakelatency is disabled, atleast thats the behavior i see now

Reply:
```cpp
config->backtrack.timeLimit = config->backtrack.fakeLatency ? config->backtrack.timeLimit : 200
```
Same as
```cpp
if (config->backtrack.fakeLatency)
    config->backtrack.timeLimit = config->backtrack.timeLimit; // or nothing to do
else
    config->backtrack.timeLimit = 200;
```

Reply:
yeah i got it to work anyway heh, its coo
see the https://github.com/danielkrupinski/Osiris/pull/1922/commits/2af7d4443f187ff892a355263c9c735d0407b81f